assets/src/scripts/store/index.js

```diff
@@ -78,7 +79,7 @@ export const Actions = {
                     const notes = collection.queryInfo[requestKey].notes;
                     const endTime = notes.requestDT;
                     const startTime = calcStartTime('P7D', endTime, 'local');
-
+                    dispatch(Actions.setRequestedDates(startTime, endTime));
```

Comment:
I find this name confusing. Could we call it setCustomDateRange?

Reply:
Sure.